docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -7,9 +7,10 @@ import TabItem from '@theme/TabItem';
 
 ## The Ziti Tunneller
 
-`ziti-edge-tunnel` is the general purpose tunneller CLI and can also run as a systemd daemon.
-
-The purpose of the tunneller is to configure host access. This means all users and all processes on the host will share the same level of access. This is accomplished by configuring the OS to have an on-board OpenZiti DNS nameserver and IP routes for authorized OpenZiti Services.
+`ziti-edge-tunnel` is a general-purpose tunneller daemon controlled by an IPC socket. The purpose of the tunneller is to
```

Comment:
Put the original wording back please. it's only a daemon depending on how you run/install it.  Leading out of the gate with"IPC socket" also adds cognitive complexity...

the original words in the lead sentence were better/correct. let's revert this